internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -59,16 +59,19 @@ func GetPluginsCLIAndRoot() (cli.CLI, *cobra.Command) {
 	c, err := cli.New(
 		cli.WithCommandName("operator-sdk"),
 		cli.WithVersion(makeVersionString()),
-		cli.WithDefaultProjectVersion(cfgv3.Version),
 		cli.WithPlugins(
 			&golangv2.Plugin{},
 			&golangv3.Plugin{},
 			&helmv1.Plugin{},
 			&ansiblev1.Plugin{},
+			&envtestv1.Plugin{},
+			&manifestsv1.Plugin{},
+			&scorecardv1.Plugin{},
 		),
-		cli.WithDefaultPlugins(cfgv2.Version, &golangv2.Plugin{}),
-		cli.WithDefaultPlugins(cfgv3.Version, &golangv3.Plugin{}),
+		cli.WithDefaultPlugins(&golangv3.Plugin{}, &manifestsv1.Plugin{}, &scorecardv1.Plugin{}),
+		cli.WithDefaultProjectVersion(cfgv3.Version),
 		cli.WithExtraCommands(commands...),
+		cli.WithCompletion,
 	)
 	if err != nil {
 		log.Fatal(err)
```

Comment:
see: 

<img width="718" alt="Screen Shot 2021-02-26 at 13 25 06" src="https://user-images.githubusercontent.com/7708031/109305786-22a8ff80-7836-11eb-839f-8ba0fcd4da0f.png">

IMO:
- The project version use ought to be `cli.WithDefaultProjectVersion(cfgv3.Version),` in the resolution
- We need to return specific help for the plugin (for this scenarios we will need to add text explaining when/how he plugin can be used - it is not part of this scope, however, we need think about)

Reply:
You are including the flag `--help` so it will always print the help.

Reply:
It is true. The problem here is that when we inform the "wrapper" plugins they shows do not have a help text added as ansible/go/helm

Reply:
Do you mean that the example should have `operator-sdk init --plugins helm -h`? (project version doesn't even make sense as they only support project version 3.

Reply:
No, IMO we need to add help explaining when/how the manifest and scorecard plugins are used. What they does, what they change and what are their purpose.

Reply:
Oh yes we can definetely improve the examples and description in each plugin.

Reply:
IMO: We can do it as a follow up. we just need to have an issue tracked for that when we are in the position to get this pr merged.